src/envo/envo-edit.owl

```diff
@@ -40206,6 +40208,38 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105013> "marine biota"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105013> <http://purl.obolibrary.org/obo/ENVO_06105012>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105014> (biodegradation)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1351%2FPAC-REC-10-12-04>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105014> "A biological process during which substance or chemical is broken down due to the action of cells."@en)
```

Comment:
@meichen-liu @raissameyer p;ease pay more attention to the grammar. 

Use:
A biological process during which a material entity is broken down.

The action of cells is not needed. It could be cell residua or exudates.